.bluemix/controller.pipeline.yml

```diff
@@ -50,9 +50,8 @@ stages:
   - name: RECOMMENDATION_PACKAGE_NAME
     value: ${RECOMMENDATION_PACKAGE_NAME}
     type: text
-  - name: OPENWHISK_AUTH
-    value: ${OPENWHISK_AUTH}
-    type: secure
+  - name: IAM_API_KEY
+    value: ${IAM_API_KEY}
```

Comment:
given this is sensitive info, we should keep the "secure" type so it stays hidden